src/lib/plugin_apis/fs.api

```diff
@@ -378,6 +378,19 @@ gboolean bd_fs_is_tech_avail (BDFSTech tech, guint64 mode, GError **error);
  */
 gboolean bd_fs_wipe (const gchar *device, gboolean all, GError **error);
 
+/**
+ * bd_fs_wipe_force:
+ * @device: the device to wipe signatures from
+ * @all: whether to wipe all (%TRUE) signatures or just the first (%FALSE) one
+ * @force: whether to force wipe even if the filesystem is mounted
+ * @error: (out): place to store error (if any)
+ *
+ * Returns: whether signatures were successfully wiped on @device or not
+ *
+ * Tech category: %BD_FS_TECH_GENERIC-%BD_FS_TECH_MODE_WIPE
+ */
+gboolean bd_fs_wipe_force (const gchar *device, gboolean all, gboolean force, GError **error);
```

Comment:
Please add a note about the potentially problematic behavior and this alternative function to the `bd_fs_wipe()` docs.

Reply:
I have added a note to `bd_fs_wipe`